tests/dft-custom-dhdf/input.dat

```diff
@@ -87,6 +93,13 @@ compare_values(-0.062529936, e_mp2, 5,  'Ne: PWPB95_PT2')   #TEST
 compare_values(-128.920706595202, edhdf, 5,  'Ne: PWPB95')   #TEST
 clean()
 
+edhdf = energy('pbe0-dh')
```

Comment:
Does it fail already for the SCF part, or at the MP2 part?
Your G16 test has frozen-core is on for the MP2 part, which is actually recommended for DHDFs. But my test has it globally off. Maybe that is the reason the test fails? The RI-MP2 error should be small enough.

Reply:
Already the DFT reference energy fails. I am a bit at a loss as to why this is. I've checked the original paper and the handy reference table from Q-Chem [1] and I think I set it up correctly, but apparently not...

[1] http://www.q-chem.com/qchem-website/manual/qchem44_manual/sect-DFT.html

Reply:
Attached out file from the current version (frozen core set to true).
[dhdf.txt](https://github.com/psi4/psi4/files/1689537/dhdf.txt)

Reply:
I used the cc-pVTZ basis set for the DHDF tests to reproduce values from a publication. The other tests used cc-pVDZ. My bad for the confusion, sorry.
PSI4 should be much closer with the correct basis ;-). If you check my PBE values in dft-custom-gga, you can see that only TM reproduces the PSI4 values to the 5th decimal. ORCA and Gaussian deviate a bit. Very sensitive to the constants used, i think.

This comparison with ORCA passes:
https://gist.github.com/hokru/fd97a2cc67404be4a8835925c06616a4

Reply:
OK I see. I need to learn how to read. The DSD-BLYP is also with FC=TRUE already, so I'll just lump them together.